app/controllers/concerns/documentation_controller_branding.rb

```diff
@@ -30,4 +30,9 @@ def plugin_documentation_url
   def wiki_url(section: '')
     documentation_url(section)
   end
+
+  # For new documentation at docs.theforeman.org
+  def docs_url(guide:, flavor:, chapter: nil)
+    ForemanThemeSatellite::Documentation::DOCS_GUIDES_LINKS.fetch(guide, {}).fetch(chapter, nil) || super
```

Comment:
I was wondering about automatically rewriting these cases by default. I think (except for real branding) the anchors should be the same. The guide name itself is (AFAIK) lowercased.

So something like this:
```suggestion
    url = ForemanThemeSatellite::Documentation::DOCS_GUIDES_LINKS.dig(guide, chapter)
    url || "#{ForemanThemeSatellite.documentation_root}/#{guide.downcase}/#{chapter}"
```

Reply:
at least today the anchors are not the same :(

Reply:
You can still override them with `DOCS_GUIDES_LINKS`. Wouldn't my suggestion still be a better fallback than the upstream links?

Reply:
fair, updated and added a test